components/questionnaire/Question.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import { Answear } from "../../interfaces/Answear";
+import { Option } from "./Option";
+
+interface QuestionProps {
+  text: string;
+  hint: string;
+  answears: number[];
+  selected: number;
+  onClick(answear: number): void;
+}
+
+export function Question(props: QuestionProps) {
+  return (
+    <div className="question">
+      <p>{props.text}</p>
+      <p className="hint">{props.hint}</p>
+      <div className="options">
+        {props.answears.map((a) => {
```

Comment:
nit: You could use more descriptive variable name instead of `a` - It would help in later coding stages.